### PR TITLE
Wheels are only universal if they support Python 2 & 3

### PR DESCRIPTION
**Pull request by michael-k**
*Issue #, if available:* n/a

*Description of changes:*

Remove the `universal` flag from section `wheel` in `setup.cfg`, because wheels are only universal if they support both Python 2 and 3.

----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kdaily** [comment]
Hi @michael-k, thanks for the PR. What's the impact of having this configured as it is now?

**Comment by michael-k** [comment]
> What's the impact of having this configured as it is now?

I'm not familiar with all the details of Python's packaging.  Therefore my answer is probably incomplete.

In general when supporting Python 2 and 3 one wanted non-universal wheels when using (optional) C extensions.  Since Python 2 is less and less common and the introduction of `python_requires` (which is used by aws-cli v1, but not v2?), the impact for Python-3-only packages is probably low.

I don't know if there's any impact at all for aws-cli v2 since it's sadly not on PyPI and people installing directly from source (see eg. https://github.com/aws/aws-cli/issues/4947#issuecomment-586250169) are creating the appropriate wheel (if at all) for their version of Python.

When running the following you can see that it creates wheels `botocore-2.0.0.dev97-py2.py3-none-any.whl` (botocore v2 also has `universal` set to `1` and doesn't use `python_requires`) and `awscli-2.1.29-py2.py3-none-any.whl`.

```sh
python -m venv .venv
source .venv/bin/activate
python -m pip install wheel
python -m pip install  https://github.com/boto/botocore/archive/v2.tar.gz
python -m pip install  https://github.com/aws/aws-cli/archive/v2.tar.gz
deactivate
```

When running `python setup.py bdist_wheel` in the v2 branch it creates `dist/awscli-2.1.29-py2.py3-none-any.whl` when `universal` is set to `1` and `dist/awscli-2.1.29-py3-none-any.whl` with the change proposed in this PR.

----

TL;DR There's probably no practical impact at the moment.

**Comment by michael-k** [comment]
Here's some documentation for universal wheels: https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels

**Comment by justindho** [comment]
It looks like the latest `setup.cfg` on the `v2` branch has this PR's proposed changes included already, so I'll be closing this PR.
